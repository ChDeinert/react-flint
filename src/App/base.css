@import "bootstrap/dist/css/bootstrap-reboot.css";

*, :before, :after {
  padding: 0;
  margin: 0;
}
