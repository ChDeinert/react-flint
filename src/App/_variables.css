:root {
  --black: #181818;
  --orange: #FF9800;
}
