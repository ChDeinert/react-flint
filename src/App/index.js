import React, { Fragment } from 'react';
import Header from 'Components/Header';
import Footer from 'Components/Footer';

import './base.css';

const App = () => (
  <Fragment>
    <Header />
    <h1>Hello World!</h1>
    <Footer />
  </Fragment>
);

export default App;
