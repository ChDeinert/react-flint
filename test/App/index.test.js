import React from 'react';
import {
  render,
} from '@testing-library/react';
import App from 'AppSource';

describe('<App />', () => {
  it('renders the Header component', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
