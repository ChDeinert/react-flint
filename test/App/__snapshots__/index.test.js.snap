// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders the Header component 1`] = `
<DocumentFragment>
  <header>
    <img
      alt="logo"
      src="logo.svg"
    />
    <span>
      react-flint
    </span>
  </header>
  <div>
    <h1>
      Welcome!
    </h1>
    <p>
      You are all set. And can start working on your Application now.
    </p>
  </div>
</DocumentFragment>
`;
